README.md

```diff
@@ -354,6 +378,7 @@ function messageCallBack(params){
   - __loaderror__: event fires when the `InAppBrowser` encounters an error loading a URL.
   - __exit__: event fires when the `InAppBrowser` window is closed.
   - __message__: event fires when the `InAppBrowser` receives a message posted from the page loaded inside the `InAppBrowser` Webview.
+  - __download__: event fires when the `InAppBrowser` loads a URL that leads in downloading of a file.
```

Comment:
Since this PR only provides an android implementation of this feature, we probably should make note of platform support.

Or we should have an implementation for the other platforms as well.